simulators/dbesm/__init__.py

```diff
@@ -250,16 +264,16 @@ def _get_status(self, params):
             return self._error(params[0], 1001)
         try:
             selected_board = next((sub for sub in self.boards
-                               if int(sub['Address']) == int(params[2])), None)
+                   if (self.boards.index(sub) + 1) == int(params[2])), None)
         except ValueError:
             return self._error(params[0], 1001)
 
         if selected_board is None:
             return self._error(params[0], 1007, params[2])
-        elif selected_board["Status"] != 0:
+        elif selected_board["Status"] == 1:
             return self._error(params[0], 1005, params[2])
         retval = self.ack + '\n'
-        retval += f'BOARD {selected_board["Address"]}\n\n'
+        retval += f'BOARD {self.boards.index(selected_board)+1}\n\n'
         retval += f'REG=[ {" ".join(map(str,selected_board["REG"]))} ]\n\n'
         retval += f'ATT=[ {"  ".join(map(str,selected_board["ATT"])) } ]'
```

Comment:
@mfioren I'm seeing two whitespaces that will separate the list of attenuations. Is this correct or we only need one?